mobile/src/main/res/values/strings.xml

```diff
@@ -122,6 +122,22 @@
     <string name="nfc_dialog_title">Write NFC tag action for this element</string>
     <string name="info_not_set">Not set</string>
 
+    <!-- Background tasks -->
+    <string name="alarm_clock">Alarm clock</string>
+    <string name="settings_alarm_clock">Send alarm clock to openHAB</string>
+    <string name="settings_alarm_clock_summary">Sends alarm clock time to a configured item</string>
+    <string name="settings_alarm_clock_item">Item</string>
+    <string name="waiting_for_network_to_send_alarm_clock">Waiting for network to send alarm clock</string>
+    <string name="notification_channel_background">Background</string>
+    <string name="notification_channel_background_description">Used to display pending background tasks</string>
+    <string name="notification_channel_background_error">Error in background tasks</string>
+    <string name="notification_channel_background_error_description">Used to notify about errors in background tasks</string>
+    <string name="error_sending_alarm_clock">Error sending alarm clock. Got HTTP error %d</string>
+    <string name="error_sending_alarm_clock_retry">Error sending alarm clock, but it\'ll be retried later</string>
```

Comment:
This text cries for 'I should not be a notification` ;-)

Reply:
Same reason as above.